locales/et.json

```diff
@@ -0,0 +1,15 @@
+{
+  "toggle_front_driver_door": "Ava/sulge eesuks",
+  "toggle_front_passenger_door": "Ava/sulge eesuks",
+  "toggle_rear_driver_door": "Ava/sulge tagauks",
+  "toggle_rear_passenger_door": "Ava/sulge tagauks",
+  "toggle_hood": "Ava/sulge kapott",
+  "toggle_trunk": "Ava/sulge pagasiruum",
```

Comment:
Surely there's something better than just "open/close".

Reply:
Not really tbh.